conflux/adapter.js

```diff
@@ -0,0 +1,88 @@
+const Web3Utils = require('web3-utils')
+const { Conflux } = require('js-conflux-sdk')
+const { Requester, Validator } = require('@chainlink/external-adapter')
+require('dotenv').config()
+
+const provider = new Conflux({
+  url: process.env.URL,
```

Comment:
You can use the `getEnv` or `getRequiredEnv` (if is required) helper to get env vars.